### ISSUE TITLE
Code text background is not themable

### ISSUE THREAD

**Issue by uloco** [CONNECT-platform/codedoc#79]
The foreground color of `` `code` `` tags is customizeable but the background color seems hardcoded. How about using a transparent default value here instead, so it just adjusts to the background? Or maybe also make it themeable ...

**Comment by loreanvictor** [comment]
yeah right now [it is bound to background of quote elements](https://github.com/CONNECT-platform/codedoc/blob/d88d2b1ae5cb8cd734d1571ea94ecea1076ffbc5/src/components/page/style.ts#L229). it should be independently theme-able while I guess falling back on quote element background is actually nice (or maybe some other good default).
